### DISCUSSION TITLE
Neural network Survival Analysis-Error

### DISCUSSION THREAD

**Discussion by drchehidayat** [mlr-org/mlr#2858] [Q&A]
Hi, 
I try to replicate the codes from the article entitles, "Neural Networks for Survival Analysis in R" by @RaphaelS1 .  However, I could not finish it due to some error. 
First of all, im using Macbook Air M1 and R software for analysis.

Here are the codes and error that I've received
Code:
install_keras(pip = TRUE, install_tensorflow = TRUE)
Error:
ERROR: Could not find a version that satisfies the requirement tensorflow (from versions: none)
ERROR: No matching distribution found for tensorflow
Error: Error installing package(s): "'tensorflow'", "'keras'"

Code: 
learners <- c(learners, lrns(c("surv.kaplan", "surv.coxph")))
design <- benchmark_grid(tasks, learners, resampling)
bm <- benchmark(design) -- ERROR
Error: Error: <TaskSurv:whas> has the following unsupported feature types: factor

I hope @RaphaelS1 or anyone can help me with this problem.

**Comment by RaphaelS1** [comment]
Hey your code seems to suggest the bug might be in survivalmodels. Would you mind reopening this discussion as an issue is RaphaelS1/survivalmodels? And when you do please can you include a `reprex` if possible with the sessionInfo included
